<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUAV_position"/>
   <addaction name="actionFitMap"/>
   <addaction name="actionDroneVisible"/>
   <addaction name="actionUAV_Pfad"/>
   <addaction name="actionPoisVisible"/>
   <addaction name="separator"/>
   <addaction name="actionGoogleMaps"/>
   <addaction name="actionDrucken"/>
   <addaction name="actionDruckansicht"/>
  </widget>
  <action name="actionUAV_position">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/myposition.png</normaloff>icons/myposition.png</iconset>
   </property>
   <property name="text">
    <string>UAV position</string>
   </property>
  </action>
  <action name="actionFitMap">
   <property name="icon">
    <iconset>
     <normaloff>icons/maximize-512.png</normaloff>icons/maximize-512.png</iconset>
   </property>
   <property name="text">
    <string>fitMap</string>
   </property>
  </action>
  <action name="actionDroneVisible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/drone-512.png</normaloff>icons/drone-512.png</iconset>
   </property>
   <property name="text">
    <string>droneVisible</string>
   </property>
  </action>
  <action name="actionUAV_Pfad">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/path2.png</normaloff>icons/path2.png</iconset>
   </property>
   <property name="text">
    <string>UAV-Pfadw</string>
   </property>
  </action>
  <action name="actionPoisVisible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/fahne.png</normaloff>icons/fahne.png</iconset>
   </property>
   <property name="text">
    <string>Pois</string>
   </property>
  </action>
  <action name="actionDrucken">
   <property name="icon">
    <iconset>
     <normaloff>icons/Printer-icon.png</normaloff>icons/Printer-icon.png</iconset>
   </property>
   <property name="text">
    <string>Drucken (Strg+p)</string>
   </property>
   <property name="toolTip">
    <string>Drucken (Strg+p)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionDruckansicht">
   <property name="icon">
    <iconset>
     <normaloff>icons/Preview-icon.png</normaloff>icons/Preview-icon.png</iconset>
   </property>
   <property name="text">
    <string>Druckansicht</string>
   </property>
  </action>
  <action name="actionGoogleMaps">
   <property name="icon">
    <iconset>
     <normaloff>icons/world_marker.jpg</normaloff>icons/world_marker.jpg</iconset>
   </property>
   <property name="text">
    <string>googleMaps</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
